docs/testing/clear-cache.md

```diff
@@ -16,7 +16,25 @@ Additionally, if you make changes to your add-in's manifest (for example, update
 
 ## Clear the Office cache on Windows
 
-To remove all sideloaded add-ins from Excel, Word, and PowerPoint, delete the contents of the folder.
+There are three ways to clear the Office cache on a Windows computer; automatically, manually, and using the Microsoft Edge developer tools. The methods are described in the following subsections.
+
+### Automatically
+
+This method is recommended for add-in development computers. If your Office on Windows version is 2108 or later, the following steps configure the Office cache to be cleared automatically every time Office is reopened.
+
+> [!NOTE]
+> The automatic method is not supported for Outlook.
+
+1. From the ribbon of any Office host, except Outlook, navigate to **File** > **Options** > **Trust Center** > **Trust Center Settings** > **Trusted Add-in Catalogs**.
+1. Enable the check box **Next time Office starts, clear all previously-started web add-ins cache**.
```

Comment:
Based on [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33535), I recommend this update:
```suggestion
1. Select the checkbox **Next time Office starts, clear all previously-started web add-ins cache**.
```